test/benchmark_min_time_flag_iters_test.cc

```diff
@@ -19,7 +19,7 @@ class TestReporter : public benchmark::ConsoleReporter {
 
   virtual void ReportRuns(const std::vector<Run>& report) BENCHMARK_OVERRIDE {
     assert(report.size() == 1);
-    iter_nums_.push_back(report[0].iterations);
+    iter_nums_.push_back(static_cast<int>(report[0].iterations));
```

Comment:
i think iter_nums_ should probably be a size_t if that's what's being returned by report[0].iterations.

Reply:
Good point. I moved further, matching exactly the type of the collection as the definition of the element is:
```cpp
typedef int64_t IterationCount;
```
So whatever will be the future underlying type of the elements: `int64_t`, `size_t` and something else, the code should still be fine.